docs/Shell-Completion.md

```diff
@@ -24,6 +24,22 @@ fi
 ```
 
 Should you later install the `bash-completion` formula, this will automatically use its initialization script to read the completions files.
+When installed, the `bash-completion` formula also runs `${HOMEBREW_PREFIX}/etc/profile.d/bash_completion.sh` and all files in the `bash_completion.d` directory. This is done by adding a line to your `.bash_profile` -
+
+```
+[[ -r "/usr/local/etc/profile.d/bash_completion.sh" ]] && . "/usr/local/etc/profile.d/bash_completion.sh"
+```
+which is mentioned in the Caveats section upon the installation of `bash-completion`
```

Comment:
No need to re-quote the suggested Caveats line; just mention that either it or the code block from earlier will work.

Reply:
Done!